app/javascript/flavours/glitch/features/ui/components/navigation_panel.jsx

```diff
@@ -19,7 +19,9 @@ const messages = defineMessages({
   notifications: { id: 'tabs_bar.notifications', defaultMessage: 'Notifications' },
   explore: { id: 'explore.title', defaultMessage: 'Explore' },
   firehose: { id: 'column.firehose', defaultMessage: 'Live feeds' },
-  direct: { id: 'navigation_bar.direct', defaultMessage: 'Private mentions' },
+  local: { id: 'navigation_bar.community_timeline', defaultMessage: 'Local' },
```

Comment:
should be 'Local timeline'

Reply:
i thought so too when writing the userscript but it was previously just "Local" and "Federated"

Reply:
can change if you'd like ofc

Reply:
oh surprising, weird that it's different from the value in `navigation_bar.community_timeline`